App/App.pro

```diff
@@ -37,7 +37,7 @@ win32 {
 CONFIG += app
 
 CONFIG += moc
-CONFIG += boost boost-serialization-lib opengl qt cairo python shiboken pyside
+CONFIG += boost boost-serialization-lib wayland opengl qt cairo python shiboken pyside
```

Comment:
Does this break the build on systems that don't have wayland?

Reply:
I will not include the changes for QMake, given the problems that rose with Qt5 and Shiboken2. Linux and BSD users that want to use Wayland should use CMake instead.